PDK_Abstraction/FinFET14nm_Mock_PDK/primitive.py

```diff
@@ -81,11 +81,12 @@ def _get_wire_terminators(intersecting_tracks):
             for j in range(self.minvias):
                 current_track = m3start + len(self._nets) * j + track
                 contacts = conn.keys()
-                if len(contacts) == 1: # Create m2 terminal
-                    i = next(iter(contacts))
-                    minx, maxx = _get_wire_terminators(conn[i])
-                    self.addWire(self.m2, net, net, i, (minx, -1), (maxx, 1))
-                else: # create m3 terminal(s)
+                #if len(contacts) == 1: # Create m2 terminal
+                i = next(iter(contacts))
+                minx, maxx = _get_wire_terminators(conn[i])
+                self.addWire(self.m2, net, net, i, (minx, -1), (maxx, 1))
+                #else: # create m3 terminal(s)
+                if len(contacts) > 1:
```

Comment:
Looks like we are always creating M2 pins even when we know we are drawing M3 pins. Is this desired functionality @arvuce22 ?

Reply:
The cell_fabric tests will need to be updated prior to merge @arvuce22.